pkg/v1/tkg/log/logger.go

```diff
@@ -98,7 +98,7 @@ func (l *logger) Outputf(format string, args ...interface{}) {
 }
 
 // V returns an InfoLogger value for a specific verbosity level.
-func (l *logger) V(level int) logr.InfoLogger {
+func (l *logger) V(level int) logr.InfoLogger { //nolint:staticcheck
```

Comment:
Why add `//nolint:staticcheck`? A comment wouldn't hurt.

Reply:
Removed the nolint comment